modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase

```diff
@@ -1389,13 +1389,37 @@ definition {
 	}
 
 	@description = "LPS-135400 - Verify it is possible to update a Relationship"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanUpdateRelationship {
 		property portal.acceptance = "true";
 
-		// TODO LPS-140590 CanUpdateRelationship pending implementation
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObject",
+			objectName_2 = "CustomObject",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
+
+		ObjectAdmin.gotoRelationshipsTab();
+
+		ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
+
+		SelectFrame(locator1 = "IFrame#IFRAME");
+
+		CreateObject.editLabel(labelName = "Relationship Updated");
+
+		Button.clickSave();
 
+		Alert.viewSuccessMessageText(successMessage = "The object relationship was updated successfully.");
```

Comment:
I was not clear on the step, my mistake. Here you need to assert that the Relationship is displayed with the new Label and not the successful message.